### PR TITLE
Stg

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
pushing the  following changes  from staging to prod:
1. How to upload certs and generate secrets
2. How to delete certs and secrets.

### CODE REVIEW

File: docs/concepts/authcodegrant.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Click Submit and Confirm to proceed the creation of the application. It may take
 1. Make sure you have logged in to [TechPass portal](http://portal.techpass.gov.sg/) as Tenant Admin or switch to Tenant (Tenant Admin) role to view all your tenant accounts.
 2. Locate the tenant account and click **Manage**.
 3. Choose **Applications** from the side menu.
-4. Locate the application for which you need to upload a new client secret and click the corresponding **Edit** icon edit-icon.
+4. Locate the application for which you need to create a new client secret and click the corresponding **Edit** icon edit-icon.
```
Comment (JLeeChan) [inline docs/concepts/authcodegrant.md:47]: is it really necessary to have the edit-icon at the end? looks redundant.
the edit-icon is not rendering the icon itself right?

Comment (SenthilKumarSivakami) [reply docs/concepts/authcodegrant.md:47]: ok noted, will remove that.

File: docs/concepts/clientcred.md [hunk 2]
```diff
@@ -117,31 +118,82 @@ Access token - use this token as Authorization bearer token with TechPass Automa
 Client Credentials Grant flow with secret is considered completed at this point.
 
 ## Establish client credentials grant flow with certificate
-### Pre-requisite
-This guide is assuming you've already have a public certificate and private key. Note: Self signed certificate are not permitted.
+### Prerequisites
+A public certificate and private key. Get a certificate issued by CoreSG (WOG CA - to be confirmed at a later time) or public CA. For example, let's encrypt or AWS certificate manager)
```
Comment (JLeeChan) [inline docs/concepts/clientcred.md:122]: `WOG CA - to be confirmed at a later time` looks like a reminder

File: docs/concepts/clientcred.md [hunk 3]
```diff
@@ -117,31 +118,82 @@ Access token - use this token as Authorization bearer token with TechPass Automa
...
+### Delete certificate
+
+As a best practice, tenant admins can delete a certifcate that is not in use or expired.
+
+**To delete a certficate**
+
+1. Open the tenant account and go to the **Edit Application** page.
+2. Go to **Certificates** section to view the list of certificates.
+3. Click the Delete icon corresponding to the certficate.
+4.  When prompted, click **Confirm**.
```
Comment (JLeeChan) [inline docs/concepts/clientcred.md:177]: extra space in front

File: docs/concepts/clientcred.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply docs/concepts/clientcred.md:122]: I will remove that info.

 However, would like to confirm if the tenant admins get the certs from CoreSG?

File: docs/concepts/clientcred.md (hunk 3 above)
Comment (SenthilKumarSivakami) [reply docs/concepts/clientcred.md:177]: Updated

File: docs/concepts/clientcred.md (hunk 2 above)
Comment (SenthilKumarSivakami) [reply docs/concepts/clientcred.md:122]: updated.

**Review by JLeeChan — APPROVED** [review]

**Pull request merged**